### DISCUSSION TITLE
maizzle: Craft beautiful HTML emails with Tailwind CSS

### DISCUSSION THREAD

**Discussion by giscus** [notiz-dev/notiz#221] [General]
# maizzle: Craft beautiful HTML emails with Tailwind CSS

Send beautiful HTML emails via NestJS crafted with Maizzle and Tailwind CSS

https://notiz.dev/blog/send-beautiful-emails-crafted-with-maizzle

**Comment by WandersonGSantos** [comment]
Best tutorial site with great tips from Nestjs, it's already great on how to send email, now with this template tip it's amazing congratulations, it helped me a lot.

**Comment by flazouh** [comment]
How do I test the appearance of my html file before sending it ?

**Reply by marcjulian** [reply]
Maizzle has a dev server run `npm run dev` in your maizzle project and visit [localhost:3000/(http://localhost:3000/) and select your mail template.
Read more in the [maizzle docs](https://maizzle.com/docs/installation#local)

**Comment by ShujaARizvi** [comment]
I tried sending the email to Gmail and Zoho with this template. But both stripped all styling. What is the solution to this?
